src/constants/v2/ballotStrategies/index.ts

```diff
@@ -21,16 +22,19 @@ export function ballotStrategies() {
       name: t`No strategy`,
       description: t`Any reconfiguration to an upcoming funding cycle will take effect once the current cycle ends. A project with no strategy may be vulnerable to being rug-pulled by its owner.`,
       address: constants.AddressZero,
+      length: 0,
```

Comment:
Adds length in ballot addresses to use for calculations in the UpcomingMessage component

Reply:
@johnnyd-eth wdyt about renaming this `durationSeconds`?

Reply:
Yeah rate